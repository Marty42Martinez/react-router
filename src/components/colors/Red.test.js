import React from 'react';
import { shallow } from 'enzyme';
import Red from './Red';

describe('Red Component', () => {
  it('renders a Red', () => {
    const wrapper = shallow(<Red />);

    expect(wrapper).toMatchSnapshot();
  });
});
