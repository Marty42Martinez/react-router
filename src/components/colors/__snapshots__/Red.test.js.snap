// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Red Component renders a Red 1`] = `
<Fragment>
  <div
    style={
      Object {
        "backgroundColor": "red",
        "height": "100vh",
        "width": "100vw",
      }
    }
  />
</Fragment>
`;
